### ISSUE TITLE
Factories with just Power Generators can't import product

### ISSUE THREAD

**Issue by samsmithnz** [satisfactory-factories/application#268]
Steps to reproduce:

1. Add a new factory for Fuel Production. 
2. Setup production to produce Fuel (any recipe, doesn't matter). 
3. Created a second factory to import the fuel and put it in power generators. 
4. Fuel doesn't appear as an import.

**Comment by Maelstromeous** [comment]
Can confirm: 
```
[{"id":1278,"name":"Fuel","products":[{"id":"LiquidFuel","amount":1,"recipe":"LiquidFuel","displayOrder":0,"requirements":{"LiquidOil":{"amount":1.5}},"buildingRequirements":{"name":"oilrefinery","amount":0.025,"powerConsumed":0.22872482000739452},"byProducts":[{"id":"PolymerResin","byProductOf":"LiquidFuel","amount":0.75}]}],"byProducts":[{"id":"PolymerResin","amount":0.75,"byProductOf":"LiquidFuel"}],"powerProducers":[],"inputs":[],"previousInputs":[],"parts":{"LiquidFuel":{"amountRequired":0,"amountRequiredExports":0,"amountRequiredProduction":0,"amountRequiredPower":0,"amountSupplied":1,"amountSuppliedViaInput":0,"amountSuppliedViaRaw":0,"amountSuppliedViaProduction":1,"amountRemaining":1,"satisfied":true,"isRaw":false,"exportable":true},"LiquidOil":{"amountRequired":1.5,"amountRequiredExports":0,"amountRequiredProduction":1.5,"amountRequiredPower":0,"amountSupplied":1.5,"amountSuppliedViaInput":0,"amountSuppliedViaRaw":1.5,"amountSuppliedViaProduction":0,"amountRemaining":0,"satisfied":true,"isRaw":true,"exportable":false},"PolymerResin":{"amountRequired":0,"amountRequiredExports":0,"amountRequiredProduction":0,"amountRequiredPower":0,"amountSupplied":0.75,"amountSuppliedViaInput":0,"amountSuppliedViaRaw":0,"amountSuppliedViaProduction":0.75,"amountRemaining":0.75,"satisfied":true,"isRaw":false,"exportable":true}},"buildingRequirements":{"oilrefinery":{"name":"oilrefinery","amount":1,"powerConsumed":0.229}},"dependencies":{"requests":{},"metrics":{}},"exportCalculator":{},"rawResources":{"LiquidOil":{"id":"LiquidOil","name":"Crude Oil","amount":1.5}},"power":{"consumed":0.229,"produced":0,"difference":-0.229},"requirementsSatisfied":true,"usingRawResourcesOnly":true,"hidden":false,"hasProblem":false,"inSync":null,"syncState":{},"displayOrder":0,"tasks":[],"notes":""},{"id":357,"name":"Generator","products":[],"byProducts":[],"powerProducers":[{"building":"generatorfuel","buildingAmount":8,"buildingCount":8,"ingredientAmount":160,"ingredients":[{"part":"LiquidFuel","perMin":160,"mwPerItem":12.5}],"powerAmount":2000,"powerProduced":2000,"recipe":"GeneratorFuel_LiquidFuel","byproduct":null,"displayOrder":0,"updated":"power"}],"inputs":[],"previousInputs":[],"parts":{"LiquidFuel":{"amountRequired":160,"amountRequiredExports":0,"amountRequiredProduction":0,"amountRequiredPower":160,"amountSupplied":0,"amountSuppliedViaInput":0,"amountSuppliedViaRaw":0,"amountSuppliedViaProduction":0,"amountRemaining":-160,"satisfied":false,"isRaw":false,"exportable":false}},"buildingRequirements":{"generatorfuel":{"name":"generatorfuel","amount":8,"powerProduced":2000}},"dependencies":{"requests":{},"metrics":{}},"exportCalculator":{},"rawResources":{},"power":{"consumed":0,"produced":2000,"difference":2000},"requirementsSatisfied":true,"usingRawResourcesOnly":false,"hidden":false,"hasProblem":false,"inSync":null,"syncState":{},"displayOrder":1,"tasks":[],"notes":""}]
```

**Comment by Maelstromeous** [comment]
Closed in #302
